app/domain/authentication/authn_k8s/inject_client_cert.rb

```diff
@@ -27,13 +27,17 @@ def validate
       end
 
       def install_signed_cert
-        exec = @kubectl_exec.new(
-          pod_name: spiffe_id.name,
-          pod_namespace: spiffe_id.namespace,
-          logger: Rails.logger,
-          kubeclient: @k8s_object_lookup.kubectl_client
+        pod_namespace = spiffe_id.namespace
+        pod_name = spiffe_id.name
+        puts "Copying SSL cert to #{pod_namespace}/#{pod_name}"
```

Comment:
What do you think about logging these instead of using `puts`?

Reply:
Hmm... haven't used the logger previously in the way we use it in the project - do we have an example I can compare/copy-paste?

Reply:
Ugh - ignore this - we use it in the other file I worked on lol

Reply:
:) it'll at least let us turn the volume up and down.